### PR TITLE
[ja] Sync up README.md (ja) to en

### PR DESCRIPTION
**Pull request by Okabe-Junya**
## Feature Description

- Sync up README.md (ja) to en
- Update some translations
- Add some notes for Japanese l10n contributor

README preview: https://github.com/kubernetes/website/blob/ff81a892bd70554b63682974511f906340358c4c/README-ja.md

### CODE REVIEW

**Comment by Okabe-Junya** [comment]
I think this PR should be reviewed by Japanese l10n team, so let me assign some Japanese reviewer / approver

/cc @kubernetes/sig-docs-ja-reviews 
/assign nasa9084

File: README-ja.md [hunk 1]
```diff
@@ -140,31 +157,40 @@ sudo launchctl load -w /Library/LaunchDaemons/limit.maxfiles.plist
...
-[コミュニティのページ](https://github.com/kubernetes/community/tree/master/sig-docs#meetings)をご覧になることで、SIG Docs Kubernetesコミュニティとの関わり方を学ぶことができます。
+[コミュニティのページ](https://github.com/kubernetes/community/tree/master/sig-docs#meetings)を確認することで、SIG Docs Kubernetesコミュニティとの関わり方を学ぶことができます。
 
 本プロジェクトのメンテナーには以下の方法で連絡することができます:
 
-- [Slack](https://kubernetes.slack.com/messages/kubernetes-docs-ja)
+- [Slack #kubernetes-docs-ja チャンネル](https://kubernetes.slack.com/messages/kubernetes-docs-ja)
 - [メーリングリスト](https://groups.google.com/forum/#!forum/kubernetes-sig-docs)
 
 ## ドキュメントに貢献する {#contributing-to-the-docs}
```
Comment (Okabe-Junya) [inline README-ja.md:167]: Screenshot 2024-05-08 at 4 26 59

Does this render well??

**Comment by Okabe-Junya** [comment]
BTW, It seems better for each l10n team to own l10n README. It would be a good idea to update the OWNERS file (using [filters](https://www.kubernetes.dev/docs/guide/owners/#filters))or relocate it to `content/<language-name>`

File: README-ja.md [hunk 2]
```diff
@@ -2,90 +2,107 @@
...
 
-2. Swagger仕様書を更新する:
+2. Swagger Specificationを更新する:
 
    ```bash
    curl 'https://raw.githubusercontent.com/kubernetes/kubernetes/master/api/openapi-spec/swagger.json' > api-ref-assets/api/swagger.json
    ```
 
-3. 新しいリリースの変更を反映するため、`api-ref-assets/config/`で`toc.yaml`と`fields.yaml`を適用する。
+3. `api-ref-assets/config/`内の`toc.yaml`と`fields.yaml`を新しいリリースの変更に合わせます。
```
Comment (inductor) [inline README-ja.md:97]: 数字の手順の部分、語調が元々揃えられてないっぽいので揃えたい（多分自分がミスった）

Comment (Okabe-Junya) [reply README-ja.md:97]: ご指摘いただいた通りですね。。 https://github.com/kubernetes/website/pull/46256/commits/7e9c5d5f570e8f13230ccdd028e59cd4e7915c1f でですます体に統一しました

File: README-ja.md (hunk 1 above)
Comment (nasa9084) [reply README-ja.md:167]: IIRC no...

**Review by nasa9084 — COMMENTED** [review]
/approve
from Japanese translation team lead's perspective

**Comment by Okabe-Junya** [comment]
Thank you all!!

**Pull request merged**